Packs/AzureSQLManagement/Integrations/AzureSQLManagement/AzureSQLManagement.py

```diff
@@ -449,6 +485,11 @@ def main() -> None:
     demisto.debug(f'Command being called is {command}')
     try:
         client = Client(
+            tenant_id=params.get('tenant_id', ''),
+            auth_type=params.get('auth_type', ''),
```

Comment:
```suggestion
            auth_type=params.get('auth_type', 'Device'),
```
I think you should set a hardcoded default here?  to preserve backward compatibility.
When a user updates to the new pack the configuration stays as before (no auth_type will be present), so we want to enforce a default not to break the integration run.

Reply:
You're right, probably missed it on this pr.. thx!